source/content/guides/integrated-composer.md

```diff
@@ -145,19 +145,18 @@ Upstream refers to the source code that is hosted in the Pantheon code repositor
      - `--no-update` tells Composer to disable automatic updates of the dependency. This makes Composer faster when adding dependencies to the Upstream as shown here.
      - `--no-update` should not be included when adding dependencies to a site.
 
-1. Set or increment the current configuration version:
+1. **Optional**. Set or increment the current configuration version. 
 
-     - If this is your first time setting the config version:
+     - This step can be skipped initially. Only perform this step if you are prompted to update the Composer config version. If this is your first time setting the config version:
```

Comment:
Can we move the "skip this initially" to the line above. This makes me think I should skid this step , but then perform the following step where I increment the version number. 

Also I think for emphasis we should use italics instead of bold - following the ref style guide https://docs.microsoft.com/en-us/style-guide/text-formatting/formatting-common-text-elements. WDYT?

```suggestion
1. _Optional_ : Set or increment the current configuration version. This step can be skipped initially. Only perform this step if you are prompted to update the Composer config version.
```